app/views/paginable/plans/_privately_visible.html.erb

```diff
@@ -77,8 +77,8 @@
                     <% end %>
                     <% role = plan.roles.where(user_id: current_user.id).first %>
                     <% conf = (role.creator? && plan.publicly_visible?) ? _("Are you sure you wish to remove this public plan? This will remove it from the Public DMPs page but any collaborators will still be able to access it.") : _("Are you sure you wish to remove this plan? Any collaborators will still be able to access it.") %>
-                    <li><%= link_to _('Remove'), deactivate_role_path(role),
-                                    method: :put, data: { confirm: conf } %></li>
+                    <li><%= link_to _('Remove'), deactivate_role_path(role), 'data-method': 'put', rel: 'nofollow',
```

Comment:
the action roles#deactivate returns HTML and there is no handle associated for ajax:success and ajax:error in any JS file that I can see in this PR. The quickest workaround is leaving as a plain action that refreshed the entire page, the elegant would be refactoring the action to return json and add handlers in JS.

Reply:
agreed that handling the entire transaction via JS would be more elegant but at this point I don't think we have time to refactor all of these Remove/Delete actions. Maybe something to work on post MVP.